Prox/Prox/Utilities/AppConstants.swift

```diff
@@ -13,7 +13,9 @@ public enum AppBuildChannel {
 public struct AppConstants {
 
     public static let isRunningTest = NSClassFromString("XCTestCase") != nil
-    public static let backgroundFetchInterval: TimeInterval = 5 * 60
+    public static let backgroundFetchInterval: TimeInterval = 1 * 60
+    public static let minimumTimeAtLocationBeforeFetchingEvents: TimeInterval = 1 * 60
```

Comment:
Yup, that is a mistake. I forgot to swap the values back to 5 & 15 minutes from when I was testing (I didn't want to wait that long for a background fetch to trigger)